EIPS/eip-inclusionlists.md

```diff
@@ -0,0 +1,170 @@
+# [DRAFT] Inclusion list EIP 
+
+eip: not assigned
+title: Inclusion lists
+description: Add an inclusion list mechanism to allow forced inclusion.
+author: mike (@michaelneuder), Terence (@terencechain), Vitalik (@vbuterin), Francesco (@fradamt), potuz (@potuz), Manav (@manav2401)
+discussions-to: not assigned
```

Comment:
please assign refs mentioned in PR description to the most current discussions-to (and mention all the other refs in that discussions-to link)

Reply:
done!